pulpcore/tests/functional/__init__.py

```diff
@@ -249,6 +249,11 @@ def pulp_api_v3_path(cli_client):
     return v3_api_root
 
 
+@pytest.fixture(scope="session")
+def pulp_api_v3_root_endpoint(pulp_cfg, pulp_api_v3_path):
```

Comment:
```suggestion
def pulp_api_v3_url(pulp_cfg, pulp_api_v3_path):
```

This is a tricky one to name, but this name made more sense to me.